runtime/src/bank.rs

```diff
@@ -7266,8 +7264,8 @@ pub(crate) mod tests {
         }
     }
 
-    fn map_to_test_bad_range() -> AccountMap<Pubkey, i8> {
-        let mut map: AccountMap<Pubkey, i8> = AccountMap::new();
+    fn map_to_test_bad_range() -> std::collections::BTreeMap<Pubkey, i8> {
+        let mut map: std::collections::BTreeMap<Pubkey, i8> = std::collections::BTreeMap::new();
         // when empty, AccountMap (= std::collections::BTreeMap) doesn't sanitize given range...
```

Comment:
this comment is now stale?

Reply:
well, still true for the btreemap, and currently AccountMap still uses Vec<BTreeMap>, so technically true...

Reply:
but, misleading. I can update it.